manual.adoc

```diff
@@ -374,22 +374,19 @@ if executable('rust-analyzer')
 endif
 ----
 
-=== Sublime Text 3
+=== Sublime Text
 
-Prerequisites: You have installed the <<rust-analyzer-language-server-binary,`rust-analyzer` binary>>.
-
-You also need the `LSP` package.
-To install it:
+==== Sublime Text 4:
+* Follow the instructions in link:https://github.com/sublimelsp/LSP-rust-analyzer[LSP-rust-analyzer].
 
-1. If you've never installed a Sublime Text package, install Package Control:
-   * Open the command palette (Win/Linux: `ctrl+shift+p`, Mac: `cmd+shift+p`)
-   * Type `Install Package Control`, press enter
-2. In the command palette, run `Package control: Install package`, and in the list that pops up, type `LSP` and press enter.
+NOTE: Install link:https://packagecontrol.io/packages/LSP-file-watcher-chokidar[LSP-file-watcher-chokidar] to enable file watching (`workspace/didChangeWatchedFiles`).
```

Comment:
Hm, this sounds like a bug in rust-analyzer, could you check that after https://github.com/rust-analyzer/rust-analyzer/pull/11151 this is not actually requried?